src/components/common-components/modal/report-issue-modal/report-issue-modal.component.tsx

```diff
@@ -70,28 +70,30 @@ const useStyles = makeStyles(() =>
       color: ScssVars.primary,
       display: "flex",
     },
-
-    spinnerScaled: {
-      marginTop: "88px",
-      marginBottom: "84px",
-      height: "100px",
-      width: "100px",
-    },
   })
 );
 
 export interface ReportIssueModalOptions {
   setOpen: (open: boolean) => void;
   open: boolean;
-  screenshot?;
   clear?: () => void;
 }
 
 export default function ReportIssueModal(options: ReportIssueModalOptions): JSX.Element {
+  const classes = useStyles();
+  const [isSent, setIsSent] = useState(false);
+  const { open, setOpen, clear } = options;
+  const [issueDescription, setIssueDescription] = useState("");
+  const title = "Zgłoś błąd";
+  const { createNotification } = useNotification();
+
+  useEffect(() => {
+    init(`${process.env.EMAIL_KEY}`);
```

Comment:
Tutaj powinno być 
```
process.env.REACT_APP_EMAIL_KEY
```

Wiem, że ty tego nie dodawałaś teraz, a po prostu przeniosłaś, ale to jest związane z tym, że my to poprawialiśmy w czasie crunch'a i to nam umknęło.

Ale z racji tego, że wysyłka maili nam działa, to zakładam, że ta metoda nie jest nam potrzebna i można ją usunąć, ponieważ my podajemy `userId` w metodzie `send` 


PS
Zmieniliśmy twoją wcześniejszą nazwę  `EMAIL_KEY`, ponieważ, w przeglądarce nie są widoczne zmienne środowiskowe które nie posiadają prefiksu `REACT_APP_` 

![image](https://user-images.githubusercontent.com/21079319/110219752-4ff24f00-7ec1-11eb-9e6f-fac93e05c19d.png)


https://medium.com/@adostes/using-environment-variables-in-a-react-application-ac3b6c307373